src/test/java/temporary/tests/TestNewCustomer.java

```diff
@@ -0,0 +1,52 @@
+package temporary.tests;
+
+import org.junit.After;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import pages.LoginPage;
+
+import static engine.drivers.WebDriverFactory.quitWebDriver;
+import static org.assertj.core.api.Assertions.assertThat;
+import static steps.MainPageSteps.*;
```

Comment:
It is a good practice not to use wildcards imports.
How to disable it in IntelliJ: https://www.baeldung.com/intellij-disable-wildcard-import

Reply:
In my branch the test is unnecessary, deleted